core/Cargo.toml

```diff
@@ -23,6 +23,7 @@ serde_yaml.workspace = true
 strum = "0.22.0"
 strum_macros = "0.22.0"
 sha3 = "0.10"
+icu_testdata = "1.2.0"
```

Comment:
maybe we should set this dependency at workspace level?

Reply:
Might be smart 👍🏻